showcase.proto

```diff
@@ -22,43 +22,44 @@ import "google/rpc/status.proto";
 
 package google.showcase.v1;
 
+
 // A service to showcase Generated API Client features and common API patterns
 // used by Google.
 service Showcase {
   // A method used to show unary methods. This method will return the message
   // that was given.
-  rpc Echo(EchoRequest) returns (EchoResponse) {}
+  rpc Echo(EchoRequest) returns (EchoResponse);
 
   // A method used to show server-side streaming methods. This method will
   // split the given content into words and will pass each word back through
   // the stream.
-  rpc Expand(ExpandRequest) returns (stream EchoResponse) {}
+  rpc Expand(ExpandRequest) returns (stream EchoResponse);
 
   // A method used to show client-side streaming methods. This method will
   // collect the contents given to it. When the stream is closed by the client,
   // this method will return the a concatenation of the strings passed to it.
-  rpc Collect(stream EchoRequest) returns (EchoResponse) {}
+  rpc Collect(stream EchoRequest) returns (EchoResponse);
 
   // A method that will be used to show bidirectional streaming methods. Upon
   // receiving a request on the stream, the same content will be passed back
   // on the stream.
 
-  rpc Chat(stream EchoRequest) returns (stream EchoResponse) {}
+  rpc Chat(stream EchoRequest) returns (stream EchoResponse);
 
   // This method is to show how a client handles a request timing out. The
   // server will wait the requested amount of response_delay and then return
   // the response specified in the request.
-  rpc Timeout(TimeoutRequest) returns (TimeoutResponse) {}
+  rpc Timeout(TimeoutRequest) returns (TimeoutResponse);
 
   // This method is used to setup the Retry method. The given list of retry behavior
   // will be bound to an ID defined by the server. Subsequent Retry requests
   // of the given ID will respond with the behavior specified in this request.
-  rpc SetupRetry(SetupRetryRequest) returns (RetryId) {}
+  rpc SetupRetry(SetupRetryRequest) returns (RetryId);
```

Comment:
Suggestion:

```protobuf
rpc CreateRetryDemonstration(CreateRetryDemonstrationRequest) returns (RetryDemonstration);
```

That allows it to expand to updating behavior in the future if necessary. (I do not see it on the horizon, but I can see how it is possible.) It also means it can follow the standard method convention.

This would also mean the next method needs to be a verb-noun (which is appropriate anyway); something like `AttemptRetryDemonstration` or `PokeRetryDemonstration`.